src/components/PetCard.js

```diff
@@ -32,11 +32,13 @@ export default function PetCard(props) {
   return (
     <div className='border rounded m-1 p-2' onClick={onClickCard}>
       <img className="img-fluid rounded" src={HOSTNAME + pet.Picture} width="100%"/>
-      <h3>
-        {pet.Name}
-        <span> </span>
-        {renderFavoriteIcon(pet.petID)}
-      </h3>
+      <div className='d-flex justify-content-between align-items-center'>
+        <h3>
+          <span className='mr-1'>{pet.Name}</span>
+          {renderFavoriteIcon(pet.PetID)}
+        </h3>
+        {pet.Distance ? <p>{pet.Distance} mi</p> : ''}
```

Comment:
This is a nice touch! No sure why, but it's only showing up on some animals

![image](https://user-images.githubusercontent.com/46770903/156234161-c66ef861-dd01-4c64-be88-4089fd91cdba.png)

Reply:
It's because only shelter #2 uses a real address, so the other pets don't have a real location. I'll update the sample data